src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -654,11 +765,41 @@ export default {
 }
 
 .conversations-search {
-	flex-grow: 1;
+	transition: all 0.3s ease;
+	width: 65%;
+	z-index: 2;
+	position : absolute;
 
 	:deep(.input-field__input) {
 		border-radius: var(--border-radius-pill);
 	}
+	&--expanded {
+		width: 90%;
+	}
+
+	&--filter{
+		pointer-events: none;
+		:deep(.input-field__input) {
+		background-color: var(--color-background-dark);
+	}
+	}
+}
+
+.options{
+	position: relative;
+	z-index: 1;
+	left : calc(65% + 10px);
+	display: flex;
+	gap: 0.25px;
+
+	&--hidden{
```

Comment:
Try to use class 'visually-hidden', it should work better

Reply:
so, `visibility: hidden;` throws out buttons from the tabbable stream, so instead of Search => Filter => Add, I jump from Search => First Chat